linuxkit/linuxkit
# Fix perms on /etc/init.d/000-issue

Signed-off-by: Avi Deitcher <avi@deitcher.net>



**- What I did**

Fix permissions to 0755 on `/etc/init.d/000-issue` in `pkg/init`.

**- How I did it**

1. `chmod 0755 pkg/init/etc/init.d/000-issue`
2. `git add` and `git commit`


**- How to verify it**

1. Make a new container image with it
2. Make a new linux kit with the new image
3. See that the whale appears

**- Description for the changelog**

Fix /etc/init.d/000-issue permissions

**- A picture of a cute animal (not mandatory but encouraged)**

**Note:** Recommend merging this before https://github.com/linuxkit/linuxkit/pull/2004 , then push, then update correct hash for that PR.
